nectar/src/config/file.rs

```diff
@@ -283,7 +289,9 @@ kraken_api_host = "https://api.kraken.com/"
 
 [maker.max_sell]
 bitcoin = 1.23456
-dai = 9876.54
+
+[maker.max_buy]
+bitcoin = 1.23456
```

Comment:
I feel like it would be more natural if the config would look something like:

[maker.btc_dai]
max_sell = "0.1"
max_buy = "0.1"

Grouping it under the trading pair makes it more obvious how this would be extended and how they relate to each other.

Reply:
Let's group when we add a second pair? :D

Reply:
More of a philosophical question I guess :D
I don't feel strongly about it although personally, I would have probably gone the extra mile here and completely changed the model to fit better in with forex :)

Reply:
I like the grouping, I already had the feeling that the `max_sell` and `max_buy` split was a bit weird, but did not invest brainpower in it yesterday. Will change it.

Reply:
Actually, on second thought, I think it depends on what we want.
The current model allows to set an overall base max limit, i.e. all BTC-XXX trading pairs would be affected. (we never buy/sell more than max bitcoin), if I change it to 

```
[maker.btc_dai]
max_sell = "0.1"
max_buy = "0.1"
```

I have to specify this limit for each trading pair. So, if we additionally have `BTC/USDT` I would have to specify:

```
[maker.btc_dai]
max_sell = "0.1"
max_buy = "0.1"

[maker.btc_usdt]
max_sell = "0.1"
max_buy = "0.1"
```

Is that what we want?

Reply:
How about:

```
[maker.btc]
max_sell = "0.1"
max_buy = "0.1"
```

For me the `max` limit was something to be defined globally within Nectar and not per trading pair. Thoughts?

Reply:
If find the per trading per one more natural but I am not a trader :grin: 
I can see the use case of specifying a complete upper bound but I am not sure that is needed because it feels like an optimization to the problem of having to duplicate configuration across several pairs. I would defer such optimizations until later or not do them at all if they would create too much of a complexity overhead. It is not like one would change this config all the time hence I think clarity is more important.

Reply:
I like your new proposals better. The only ambiguuty here is that it might not be clear what the amount refer to. I would try to introduce "BTC" or "base" in the config terminology.

Reply:
hm... I am a bit divided here. I find the fact, that one cannot see `bitcoin` not perfect neither. It would be good to see that the value specified is in the base currency. I know that the currency pair would imply it but I still think that it would be nice to have the `bitcoin` there so it is absolutely clear. 
Furthermore, I would not invest time into changing this now. It is a refactoring on top of a nice to have feature... So I tend to roll with @D4nte 's initial suggestion to tackle this once we add another pair.

Reply:
> hm... I am a bit divided here. I find the fact, that one cannot see bitcoin not perfect neither. It would be good to see that the value specified is in the base currency. I know that the currency pair would imply it but I still think that it would be nice to have the bitcoin there so it is absolutely clear.

You could say `max_sell_quantity` and `max_buy_quantity` in addition to the currency pair to make it clear that we are talking about the quantity and not the price.

> Furthermore, I would not invest time into changing this now. It is a refactoring on top of a nice to have feature... So I tend to roll with @D4nte 's initial suggestion to tackle this once we add another pair.

The more I think about it the more I would want to change the config because the current one is super confusing :see_no_evil: